### ISSUE TITLE
Working together on the R-Instat help?

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat-Help#28]
I have started work on the R-Instat help file.  It is now at the stage where I suggest a combined effort would be very useful.  However I am unsure exactly how we should do this.  This message is first to help us to get a strategy and then to proceed with the work.

Strategy first.  
1. **Software:**  We seem to have decided on HelpNdoc.  I have been using it and have been pretty happy, so don't see that we need to change.  Anyone can download it free of charge and use it forever.  So we can each have a copy.  However the free copy leaves a publicity message at the bottom of each help page.  To avoid this I have paid for my own copy of the Standard Version.  This does not leave the message.
2. So, one way is for us each to have our own version of helpNdoc.  We work in turns using the software. We share the un-compiled version of the file on github.  Then I produce the compiled version, so it doesn't have their publicity.  That's a bit like producing the next version of the R-Instat software, where Maxwell produces the latest compiled version.
3. Unlike R-Instat, it is very easy to produce the compiled version.  So we can each produce one anytime, to check all is working as we would like, e.g. hyperlinks, etc.  
4. But it would be nicer to work in parallel, rather than taking turns with updating the HelpNDoc file.  I don't see how to do that easily, with the software.
5. So here is my suggestion for the way we work.  This is for discussion, not just to accept.
a) We capture figures (mainly dialogues) in a way we can edit and add comments, and then insert.  I have been doing this with powerpoint and it works well.  We can have all the figures for a section of the help in a single file, which we can share over github.
b) We each have a copy of HelpNDoc
c) Others use Word, while one person is using and updating the HelpNDoc file.  They are also shared on github.
d) The word file(s) are imported into HelpNDoc, by whoever is using HelpNDoc at the time.
e) When needed, then I'll produce the compiled version for the next release.

By the way, on general layout I quite like the [HelpNDoc help on itself](https://www.helpndoc.com/sites/default/files/documentation/html/Overviewoftheuserinterface.html)!  Simple but clear.

What do you think?

**Comment by dannyparsons** [comment]
I agree that only one person should work on the HelpNDoc file at a time, otherwise it's going to get messy with conflicting versions. Others could look at the file through HelpNDoc but not make changes that are saved.

If the intermediate work is done in Word and PowerPoint then should these be on Dropbox instead? So that they can be worked on simultaneously online.
